airbyte-integrations/connectors/source-exchange-rates-tutorial/unit_tests/test_incremental_streams.py

```diff
@@ -0,0 +1,59 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+
+from airbyte_cdk.models import SyncMode
+from pytest import fixture
+from source_exchange_rates_tutorial.source import IncrementalExchangeRatesTutorialStream
+
+
+@fixture
+def patch_incremental_base_class(mocker):
+    # Mock abstract methods to enable instantiating abstract class
+    mocker.patch.object(IncrementalExchangeRatesTutorialStream, "path", "v0/example_endpoint")
+    mocker.patch.object(IncrementalExchangeRatesTutorialStream, "primary_key", "test_primary_key")
+    mocker.patch.object(IncrementalExchangeRatesTutorialStream, "__abstractmethods__", set())
+
+
+def test_cursor_field(patch_incremental_base_class):
+    stream = IncrementalExchangeRatesTutorialStream()
+    # TODO: replace this with your expected cursor field
+    expected_cursor_field = []
+    assert stream.cursor_field == expected_cursor_field
```

Comment:
how does it work?

that's opposite to what is implemented [here](https://github.com/airbytehq/airbyte/pull/14442/files#diff-b552061aa2b8e7c43d45df2a942450a1ca98515c69142f44a09414b377fb47fbR38)